test/modules/core/shaderlib/project/project-glsl.spec.js

```diff
@@ -62,6 +64,67 @@ const TEST_VIEWPORT_ORTHO = new Viewport({
   })
 });
 
+const DUMMY_SOURCE_BUFFER = new Buffer(gl, 1);
+const OUT_BUFFER = new Buffer(gl, 16);
+
+// used in printing a float into GLSL code, 1 will be 1.0 to avoid GLSL compile errors
+const MAX_FRACTION_DIGITS = 5;
+
+const TRANSFORM_VS = {
+  project_scale: meter => `\
+varying float outValue;
+
+void main()
+{
+  outValue = project_scale(${meter.toFixed(MAX_FRACTION_DIGITS)});
+}
+`,
+  project_scale_vec2: meter => `\
+  varying vec2 outValue;
+
+  void main()
+  {
+    outValue = project_scale(vec2(${meter[0].toFixed(MAX_FRACTION_DIGITS)}, ${meter[1].toFixed(
+    MAX_FRACTION_DIGITS
+  )}));
+  }
+  `,
+  project_scale_vec3: meter => `\
+  varying vec3 outValue;
+
+  void main()
+  {
+    outValue = project_scale(vec3(${meter[0].toFixed(MAX_FRACTION_DIGITS)}, ${meter[1].toFixed(
+    MAX_FRACTION_DIGITS
+  )}, ${meter[2].toFixed(MAX_FRACTION_DIGITS)}));
+  }
+  `,
+  project_position: pos => `\
+varying vec4 outValue;
+
+void main()
+{
+  outValue = project_position(vec4(${pos[0].toFixed(MAX_FRACTION_DIGITS)}, ${pos[1].toFixed(
+    MAX_FRACTION_DIGITS
+  )}, ${pos[2].toFixed(MAX_FRACTION_DIGITS)}, ${pos[3].toFixed(
+    MAX_FRACTION_DIGITS
+  )}), vec2(0., 0.));
```

Comment:
Take advantage of the xy64Low argument to improve precision

Reply:
Added code to use low64 bits, but LNGLAT mode test still fails, as these not used in base project module. We can remove the tolerance for project-64 module tests.